### PR TITLE
Update OpenTasksPal to use latest ContentPal with Projection, View ch…

### PR DESCRIPTION
**Pull request by lemonboston**
…anges. #517

### CODE REVIEW

**Comment by lemonboston** [comment]
Build fails because `TaskProviderTest` doesn't compile.
The project builds locally for me, so it looks like that compiling androidTest classes is not part of the default build task (`gradlew build`).
image

I am going to fix this now.

**Comment by lemonboston** [comment]
Travis build have failed twice now with this:
```
Skipping device 'test(AVD)' for 'opentasks:': Unknown API Level

 > : No compatible devices connected.[TestRunner] [31mFAILED [0m
Found 1 connected device(s), 0 of which were compatible.
:opentasks:connectedDebugAndroidTest FAILED

FAILURE: Build failed with an exception.
```
Not sure how to fix this. It seems it can't determine the API Level of the emulator. And only sometimes?
I am going to restart the build for a 3rd time now.

**Comment by dmfs** [comment]
I already created an issue for that: #516

**Comment by lemonboston** [comment]
All checks pass here now, ready for review.

File: opentaskspal/src/main/java/org/dmfs/opentaskspal/tables/TaskListScoped.java [hunk 1]
```diff
@@ -83,11 +83,9 @@ public TaskListScoped(@NonNull RowSnapshot<TaskContract.TaskLists> taskListRow,
     }
 
 
-    @NonNull
-    @Override
-    public View<TaskContract.Tasks> view(@NonNull ContentProviderClient client, @NonNull String... projection)
+    public View<TaskContract.Tasks> view(@NonNull ContentProviderClient client)
```
Comment (dmfs) [inline opentaskspal/src/main/java/org/dmfs/opentaskspal/tables/TaskListScoped.java]: Where did the annotations go?

**Pull request merged**